read-the-docs/2/Makefile

```diff
@@ -21,7 +21,7 @@ help:
 # auto build documentation on file change and provide web browser "make livehtml"
 # --poll for windows. virtual box does not notify file changes.
```

Comment:
do we not need the --poll for windows?

Reply:
on my machine it fail because this param is not known...
And it's nether documented: https://pypi.org/project/sphinx-autobuild/

So I'm not really sure why this was in charge or for what it was needed

Reply:
ok. it's fine for me. Maybe @alexsuter  can contribute to the discussion after his vacation

Reply:
in earlier versions of sphinx-autobuild there was a parameter "--poll". see https://pypi.org/project/sphinx-autobuild/0.7.1/ and was necessary that this runs on windows.

but seems to be removed now -> https://github.com/executablebooks/sphinx-autobuild/blob/master/NEWS.rst

please also delete the comment "# --poll for windows. virtual box does not notify file changes."